src/components/plots/plot.ts

```diff
@@ -323,6 +329,21 @@ module Plottable {
       return d3.merge(allSetsOfExtents);
     }
 
+    protected _makeFilterByAttr(attr: string): _Accessor {
+      return (datum: any, index: number, dataset: Dataset, metadata: Plots.PlotMetadata) => {
+        var attrBinding = this._attrBindings.get(attr);
+        if (attrBinding != null) {
+          var accessor = attrBinding.accessor;
+          var scale = attrBinding.scale;
+          if (scale != null) {
+            var range = scale.range();
+            return Utils.Methods.inRange(scale.scale(accessor(datum, index, dataset, metadata)), range[0], range[1]);
+          }
+        }
+        return true;
+      };
+    }
```

Comment:
Same comment about the instance variables ties to this implementation.  Also, it is nice that it works for all attributes, but does it make sense to?